extensions/ql-vscode/src/run-queries-shared.ts

```diff
@@ -150,7 +151,9 @@ export class QueryEvaluationInfo extends QueryOutputDir {
     };
   }
   getSortedResultSetPath(resultSetName: string) {
-    return join(this.querySaveDir, `sortedResults-${resultSetName}.bqrs`);
+    const hasher = new MurmurHash3.default();
```

Comment:
Is there a specific reason for choosing to use MurmurHash here? I would prefer to use a hashing algorithm for which we do not need a dependency, so something like SHA256 using [`crypto.createHash`](https://nodejs.org/docs/latest-v18.x/api/crypto.html#cryptocreatehashalgorithm-options) would be just as good IMO. I don't think the performance of this method matters, so using a more expensive hashing algorithm shouldn't really be a problem.

Reply:
I saw that `imurmurhash` was already a dependency, but didn't notice it was only a dev dependency. I agree the performance difference shouldn't matter in this case and will update the PR to use SHA256 using `crypto.createHash` instead.